### PR TITLE
Fix 502 error on warning API

### PR DESCRIPTION
**Pull request by faizpuru**
Use the new API to request warnings

### CODE REVIEW

File: tests/test_warning.py [hunk 1]
```diff
@@ -35,23 +34,22 @@ def test_currentphenomenons(requests_mock: Mock) -> None:
 
     current_phenomenoms = client.get_warning_current_phenomenoms(domain="32", depth=1)
 
-    assert type(current_phenomenoms.update_time) == int
-    assert type(current_phenomenoms.end_validity_time) == int
-    assert type(current_phenomenoms.domain_id) == str
+    assert isinstance(current_phenomenoms.update_time, int)
+    assert isinstance(current_phenomenoms.end_validity_time, int)
+    assert isinstance(current_phenomenoms.domain_id, str)
```
Comment (Quentame) [inline tests/test_warning.py:39]: Changing type to isinstance could be done everywhere in one separate PR, same for #679

File: tests/test_warning.py [hunk 2]
```diff
@@ -86,13 +83,12 @@ def test_currentphenomenons_with_coastal_bulletin(dep: str, res: bool) -> None:
         domain=dep, depth=1, with_costal_bulletin=True
     )
     has_coastal_phenomenom = any(
-        phenomenom["phenomenon_id"] == 9
+        phenomenom["phenomenon_id"] == "9"
```
Comment (Quentame) [inline tests/test_warning.py:86]: mypy issue here, maybe still an int ?

File: tests/test_warning.py [hunk 3]
```diff
@@ -101,7 +97,7 @@ def test_full_with_coastal_bulletint(dep: str, res: bool) -> None:
     full_phenomenoms = client.get_warning_full(domain=dep, with_costal_bulletin=True)
 
     has_coastal_phenomenom = any(
-        phenomenom["phenomenon_id"] == 9
+        phenomenom["phenomenon_id"] == "9"
```
Comment (Quentame) [inline tests/test_warning.py:100]: mypy issue here, maybe still an int ?

**Review by Quentame — COMMENTED** [review]
Thanks 🎉 
The fix is only this, cool !
Did not have time to check before.

**Comment by faizpuru** [comment]
I updated my code. The phenomenon id returned by meteo france is now a character string. There should be no more typing errors.

File: tests/test_warning.py (hunk 3 above)
Comment (faizpuru) [reply tests/test_warning.py:100]: I updated my code. The phenomenon id returned by meteo france is now a character string. There should be no more typing errors.

File: tests/test_warning.py (hunk 2 above)
Comment (faizpuru) [reply tests/test_warning.py:86]: I updated my code. The phenomenon id returned by meteo france is now a character string. There should be no more typing errors.

**Comment by faizpuru** [comment]
Any news ?

File: src/meteofrance_api/const.py [hunk 4]
```diff
@@ -16,31 +16,31 @@
...
+    "1": "Wind",
+    "2": "Rain-Flood",
+    "3": "Thunderstorms",
+    "4": "Flood",
+    "5": "Snow/Ice",
+    "6": "Extreme high temperature",
+    "7": "Extreme low temperature",
+    "8": "Avalanches",
+    "9": "Coastal Event",
+}
```
Comment (Quentame) [inline src/meteofrance_api/const.py:43]: On the TODO list: We should use the Warning dictionary API instead of hardcoded values, see #682

**Review by Quentame — APPROVED** [review]

**Pull request merged**